src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -162,7 +162,7 @@ bool ExtractArchive (string url, string file, string contentDir)
 					LogMessage ("Extracting {0} to {1}", file, contentDir);
 					using (var zip = MonoAndroidHelper.ReadZipFile (file)) {
 						int extracted = 0;
-						var o = Math.Max(1, (zip.Entries.Count / 10));
+						var o = Math.Max(1, (zip.NumberOfEntries / 10));
```

Comment:
We should keep track of the libZipSharp API suggestions... Should this be `EntryCount`?